test api/REST Client/AdminCenter API/Environments database operations.http

```diff
@@ -0,0 +1,60 @@
+@clientid = 7e930f8e-fac4-43a6-a242-3a4079ac4553
+@clientsecret = clientSecret
+@scope = https://api.businesscentral.dynamics.com/.default
+@tenant = ffb102fa-38b1-4d6b-ad59-1192df615c1e
+@environmentName = Sandbox1
+@aPIPublisher = ciellos
+@aPIGroup = salesOrderApi
+@aPIVersion = v1.0
+@baseAPIUrl = {{baseUrl}}/v2.0/{{environmentName}}/api/{{aPIPublisher}}/{{aPIGroup}}/{{aPIVersion}}
```

Comment:
I think the base URL for this app is not right. Have you tried to execute the HTTP requests below ?

Reply:
Since you are using : @baseUrl property only i think we can remove baseAPIUrl

Reply:
@erjonkuka96, first of all, thanks for all the insights.

Code reviewed. As suggested I removed baseAPIUrl and variables related to it.

Yes, I've tried. I got the url from here: https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/administration/administration-center-api_environment_database_export.
I'm not sure but I think the problem is in the token request because I used the same endpoint on Postman and it worked. I've inspected/decoded the tokens that I get from Postman to the ones I get from REST and comparing them they're different. Also, if I copy the token that I get from Postman and use it in the http request it works. 
However, I'm not finding how to change the token request in REST to get the "same token structure" as I get on Postman...

Thanks again.